src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/hmi/rc_get_capabilities_response.h

```diff
@@ -66,6 +66,8 @@ class RCGetCapabilitiesResponse : public app_mngr::commands::ResponseFromHMI {
    **/
   virtual void Run();
 
+  virtual void onTimeOut();
```

Comment:
@ydementieiev Please, don't use the `virtual` keyword. Use the `override` keyword.

Reply:
@IGapchuk done